nvtabular/ops.py

```diff
@@ -787,6 +794,23 @@ def apply_mean_std(self, gdf, stats_context, cont_names):
                 new_gdf[new_col] = new_gdf[new_col].astype("float32")
         return new_gdf
 
+    def apply_min_max(self, gdf, stats_context, cont_names):
+        # calculate min and max
+        min_max = {}
+        min_max["min"] = gdf[cont_names].min()
+        min_max["max"] = gdf[cont_names].max()
```

Comment:
The 'gdf' here isn't the full dataset - its just a piece thats loaded up into GPU memory. As such we can't calculate the min/max values here and need to do in a separate StatOperator

We do have a 'MinMax' op already that will calculate these statistics across the entire dataframe though: https://github.com/NVIDIA/NVTabular/blob/ddf58ecc8603ec09129cad420adf9916faa928c8/nvtabular/ops.py#L212-L253 . 

You should be able to use in the same way as we do with the Moments (getting values through the stats_context).